### PR TITLE
Add bug report template

### PR DESCRIPTION
**Pull request by DhritiShikhar**
### Motivation

This PR adds a bug report template for the service-binding-operator project.

### Changes

This PR creates a file `bug_report.md` under `.github/ISSUE_TEMPLATE` folder.

### CODE REVIEW

**Comment by jasperchui** [comment]
Does it make sense to include a request for logs and/or screenshots to help us investigate the issue?  Thanks.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
```
Comment (pedjak) [inline .github/ISSUE_TEMPLATE/bug_report.md:4]: it is redundant to have this prefix, given `kind/bug` label we add.

Comment (DhritiShikhar) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Removed the title

**Comment by DhritiShikhar** [comment]
> Does it make sense to include a request for logs and/or screenshots to help us investigate the issue? Thanks.

Added those two sub-titles in the template.. Please review again @jasperchui

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (DhritiShikhar) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Please review again @pedjak

**Review by pedjak — APPROVED** [review]

**Comment by pedjak** [comment]
please sign off your commits.

**Comment by DhritiShikhar** [comment]
> please sign off your commits.

Done

**Pull request merged**